bigquery/bigquery_datatransfer_quickstart/main.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2017 Google LLC. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START bigquery_datatransfer_quickstart]
+// Sample bigquery-quickstart creates a Google BigQuery dataset.
+package main
+
+import (
+	"fmt"
+	"log"
+
+	// Imports the BigQuery Data Transfer client package.
+	"cloud.google.com/go/bigquery/datatransfer/apiv1"
+	"golang.org/x/net/context"
+	"google.golang.org/api/iterator"
+	datatransferpb "google.golang.org/genproto/googleapis/cloud/bigquery/datatransfer/v1"
+)
+
+func main() {
+	ctx := context.Background()
+
+	// Sets your Google Cloud Platform project ID.
+	projectID := "YOUR_PROJECT_ID"
+
+	// Creates a client.
+	client, err := datatransfer.NewClient(ctx)
+	if err != nil {
+		log.Fatalf("Failed to create client: %v", err)
+	}
+
+	req := &datatransferpb.ListDataSourcesRequest{
+		Parent: fmt.Sprintf("projects/%s", projectID),
+	}
+	it := client.ListDataSources(ctx, req)
+	fmt.Println("Supported Data Sources:")
+	for {
```

Comment:
Not sure if it makes it more or less readable, but you could squish these 4 lines and pop them into the for loop for a more iterator-y feeling: `for ds, err := it.Next(); err != iterator.Done; ds, err = it.Next() {`.

I'm unsure which style these docs prefer (noob here heh).

Reply:
less readable for sure. I was also going to comment with this but searched the repo. it looks like for { if err != ... } is the preferred idiom currently.

Reply:
👍